src/stories/Components/Tag/Tag.mdx

```diff
@@ -0,0 +1,125 @@
+import { Meta, Story, Canvas, Controls } from '@storybook/blocks'
+import * as TagStories from './Tag.stories'
+
+import { createAlert } from '../Alert/Alert.js'
+
+<Meta of={TagStories} />
+
+# Tag
+
+Tags are for interaction and input.
+
+## Playground
+
+<Canvas of={TagStories.Default} />
+<Controls of={TagStories.Default} />
+
+### Selective Imports
+
+Tag can be used as a standalone component, however, it does have a few dependencies. To see a full dependency list, see [Dependency Graph](?path=/docs/overview-selective-imports--docs#dependencies)
+
+```css
+// Depends on
+@import '@rolemodel/optics/dist/scss/core/tokens';
+@import '@rolemodel/optics/dist/scss/core/base';
+@import '@rolemodel/optics/dist/scss/components/icon';
+
+// Component
+@import '@rolemodel/optics/dist/scss/components/tag';
+```
+
+## Variations
+
+### Default
```

Comment:
I typically add a blurb explaining what class is used to accomplish this variation, and also will specify if its a modifier.